### PR TITLE
Push qcow2 image to artifacts every build

### PR DESCRIPTION
**Pull request by johnbieren**
We want to push every qcow2 we create to a directory. These directories should rotate out every X days, which I set to 3 for now; can easily be changed. 

Signed-off-by: Johnny Bieren <jbieren@redhat.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -271,6 +271,17 @@ podTemplate(name: podName,
...
                         // We always run, but don't always push to artifacts
                         env.PUSH_IMAGE = "false"
+                        // Check if we should wipe daily image dir
+                        env.numImageDirs = 3
+                        env.dailyImageDir = pipelineUtils.getDailyImageDir(numImageDirs)
+                        env.wipeBool = pipelineUtils.checkDailyImageDir(numImageDirs)
+                        if (env.wipeBool) {
+                            // Wipe image dir
+                            env.rsync_paths = "."
+                            env.rsync_from = "\$(mktemp -d)/"
```
Comment (scoheb) [inline Jenkinsfile]: hmmm...not sure this is going to work...bash is not executed here. you would have to do the 'sh' and get the stdout and somehow parse the new dir name.

**Comment by scoheb** [comment]
I lot of your code might have sandbox restriction errors...

File: Jenkinsfile (hunk 1 above)
Comment (johnbieren) [reply Jenkinsfile]: Yea... a lot of finger crossing going on. Maybe I'll just add a .sh file to a container to do all this for me and just call that

**Comment by johnbieren** [comment]
[test] wget pagure error

**Comment by johnbieren** [comment]
[test] whitelist function

**Comment by johnbieren** [comment]
[test] wget pagure failiure

**Comment by johnbieren** [comment]
[test] wget pagure failiure

**Comment by scoheb** [comment]
lgtm

**Comment by johnbieren** [comment]
[test]

**Comment by johnbieren** [comment]
Closing this in favor of opening a new PR that reuses the logic implemented to archive the logs for the image builds.

**Pull request closed without merging**